examples/aws_lambda/aws_lambda.py

```diff
@@ -31,5 +31,5 @@ def handler(event, context):
 # export SLACK_BOT_TOKEN=xoxb-***
```

Comment:
@zimeg quick question: our internal report mentioned the necessity to rename this file. when you did testing for this change, was it unnecessary?

Reply:
@seratch Thanks for checking on this! I noticed that [updating the `role`](https://github.com/slackapi/bolt-python/pull/1224/files#diff-d2e9b108571fc7c7b9f77bf73bcb9ec43e03d2beda493cbd0741201055cbc369L7) used by `deploy.sh` was enough to resolve errors, but I will check this once more 🔍 ✨